examples/nextjs-tiptap/app/api/database.ts

```diff
@@ -1,63 +1,71 @@
 // A mock database with example users
 const USER_INFO: Liveblocks["UserMeta"][] = [
   {
-    id: "charlie.layne@example.com",
+    // TODO: Revert to email
```

Comment:
@marcbouchenoire Did you mean to revert these as part of this PR, or later?

Reply:
Later, the backend needs more changes for this I think (https://liveblocks.slack.com/archives/C07VAF08PAN/p1737411180057969).